crates/optimism/payload/src/traits.rs

```diff
@@ -27,3 +34,150 @@ where
 {
     type _TX = Tx;
 }
+
+/// Wrapper struct that allows to convert `BestTransactions` (used in tx pool) to
+/// `PayloadTransactions` (used in block composition).
+#[derive(Debug)]
+pub struct BestPayloadTransactions<T, I>
```

Comment:
```suggestion
pub struct OpBestPayloadTransactions<T, I>
```
although not sure if this type is necessary..see comment below

Reply:
This type only used by OP as i see

Reply:
yes but there is an equivalent l1 type with the same name